src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key in action.keysToRemove) {
```

Comment:
```suggestion
        for (const key of action.keysToRemove) {
```
keysToRemove - це масив. Має бути цикл for of